ENIGMAsystem/SHELL/Graphics_Systems/General/GSstdraw.cpp

```diff
@@ -517,94 +492,68 @@ void draw_roundrect_ext(gs_scalar x1, gs_scalar y1, gs_scalar x2, gs_scalar y2,
     gs_scalar t=x2;
     x2=x1;
     x1=t;
-  }
+    }
   if (y1>y2) {
     gs_scalar t=y2;
     y2=y1;
     y1=t;
   }
-  x1++, y1++; //This fixes an off-by-one error when drawing over a regular draw_rectangle
```

Comment:
No, if you look at the screenshots the edge of the rounded rectangle is perfectly aligned with the rectangle behind it and has no 'bump' between the rounded corners and edges, you'll see these if you add 1 to my .24, .49 etc or if you remove the main branch's x++, y++ thing